packages/macros/src/macros-config.ts

```diff
@@ -258,8 +270,9 @@ export default class MacrosConfig {
   // normal node_modules resolution can find their dependencies. In other words,
   // owningPackageRoot is needed when you use this inside classic ember-cli, and
   // it's not appropriate inside embroider.
-  babelPluginConfig(owningPackageRoot?: string): PluginItem[] {
+  babelPluginConfig(appOrAddonInstance?: any): PluginItem[] {
     let self = this;
+    let owningPackageRoot = appOrAddonInstance ? appOrAddonInstance.root || appOrAddonInstance.project.root : null;
```

Comment:
Looks like this introduces a build error:

```
Error: packages/macros/src/macros-config.ts(275,108): error TS1005: ':' expected.
```

I think this might work:

```suggestion
    let owningPackageRoot = appOrAddonInstance?.root ?? null;
```